/test_robot/pwm_driver_node:
  ros__parameters:
    pwm_frequency: 47.2
    pwm_ms_bias: 0.09
    thruster_num: 4
    thruster_ch_list: [1, 2, 3, 4]
    thruster_topic_list: [thruster_1, thruster_2, thruster_3, thruster_4]
    thruster_min_us: [1100, 1100, 1100, 1100]
    thruster_max_us: [1900, 1900, 1900, 1900]